aiohttp/worker.py

```diff
@@ -55,6 +57,11 @@ def run(self) -> None:
             self.loop.run_until_complete(self._task)
         except Exception:
             self.log.exception("Exception in gunicorn worker")
+            if self.started:
+                self.exit_code = 1
+            else:
+                self.exit_code = Arbiter.APP_LOAD_ERROR
```

Comment:
```suggestion
            self.exit_code = 1 if if self.started else Arbiter.APP_LOAD_ERROR
```

Personally, I think this is a little cleaner, avoiding the nesting.

Reply:
I agree. But I didn't which syntax is preferred in aiohttp project. That's why I chose that. But of course I will update it.